api.badboy.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Bad Boy's Api",
+  "repo": "https://github.com/Bad-Boy-Codes/bad-boy-codes.github.io",
+  "owner": {
+    "username": "badboy",
```

username is wrong ig
```suggestion
    "username": "Bad-Boy-Codes",
```